### PR TITLE
added a share solves options dropdown

### PR DESCRIPTION
**Pull request by syrok94**
**What does this PR do?**
feat #324 

**Related Issue(s)**
none

**Screenshots or GIFs (if applicable)**
a
b

**Additional Comments**
hii @bryanlundberg , please review and suggest if some optimization can be made

**By submitting this PR, I confirm that:**
- [x] I have reviewed my code and believe it is ready for merging.
- [x] I understand that this PR may be subject to review and changes.
- [x] I agree to abide by the code of conduct and contributing guidelines of this project.

### CODE REVIEW

File: src/components/solves/ShareMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  const handleShareAo12 = async () => {
+    setShareSolveModal(false);
+    if ("clipboard" in navigator) {
+      const data = `Generated by NexusTimer : ${date} \nAverage : ${formatedAo12} \nList of Times : \n ${[
+        ...formattedLast12Solves,
+      ]}`;
+      await navigator.clipboard.writeText(data);
+    }
+  };
```
Comment (bryanlundberg) [inline src/components/solves/ShareMenu.tsx:44]: It's not required to include a comma (",") after each

**Review by bryanlundberg — CHANGES_REQUESTED** [review]
I believe your initial intention was to keep the first menu open, and the second menu opens on top of the previous menu.

However, it's a bit strange because the animations are activated, and in the background, I can see the previous menu. It doesn't give the impression of being the same.

**Comment by syrok94** [comment]
hii @bryanlundberg  I have made changes and please let me know more info regarding the previous menu in the background.

**Comment by bryanlundberg** [comment]
I like the latest changes; let's keep that design.👌

Regarding the clipboard result, it's important to keep it compact without empty rows.

image

Also, please ensure that the left side remains aligned.

image

**Comment by syrok94** [comment]
Hi @bryanlundberg, Fix the changes, please review.

**Comment by bryanlundberg** [comment]
You are doing great, but I have a few minor suggestions 😬

Last changes:

- "Generated by NexusTimer" is missing internalization.
- There should not be a space before the colon (":").

Additionally, this was not request at start but I think it helps improve the visual layout. If you could numerate the solves like this:

1. [time]: [scramble]
2. [time]: [scramble]
3. [time]: [scramble]
... 


Please, @syrok94 😅

**Comment by syrok94** [comment]
@bryanlundberg Fixed it.
> Generated by NexusTimer: 7/15/2024 
> Average 5: 0.67 
>
>List of times: 
>
>   1. (2.04) F  U2 R  U2 R  U  R2 
>   2. 0.62 U' R2 U2 F  U' F' U  R2 
>   3. (0.50) U' F2 R2 U' R  U' F' R2 
>   4. 0.54 U' R  U  F' U  R' U  R' U' 
>   5. 0.84 U2 R' F' R2 U  F' R' F  U2

**Comment by bryanlundberg** [comment]
> @bryanlundberg Fixed the changes and I want to know that you want to Internationalize the "Generated by NexusTimer" , "Average" and "List of times" ?
> 
> > Generated by NexusTimer: 7/15/2024
> > Average 5: 0.67
> > List of times:
> > 
> > 1. (2.04) F  U2 R  U2 R  U  R2
> > 2. 0.62 U' R2 U2 F  U' F' U  R2
> > 3. (0.50) U' F2 R2 U' R  U' F' R2
> > 4. 0.54 U' R  U  F' U  R' U  R' U'
> > 5. 0.84 U2 R' F' R2 U  F' R' F  U2

That's correct, let's internalize everything!

**Comment by syrok94** [comment]
@bryanlundberg fixed it

**Comment by bryanlundberg** [comment]
Feature ready!
Thank you again @syrok94 👌

**Pull request merged**